gemrb/core/PathFinder.h

```diff
@@ -45,7 +45,8 @@ enum class PathMapFlags : uint8_t {
 	DOOR_IMPASSABLE = 32,
 	PC = 64,
 	NPC = 128,
-	ACTOR = (PC | NPC),
+	//ACTOR = (PC | NPC),
+	ACTOR = PC,
```

Comment:
With the current implementation, where we only set the NPC bit and figure out bumpability elsewhere (since it's complicated), there's no need for such a bit. Just set ACTOR to 64 and rename NPC to whatever is sensible. And then remove the two places where it is set.